src/banking_stage.rs

```diff
@@ -79,7 +79,12 @@ impl BankingStage {
     ) -> Result<()> {
         let timer = Duration::new(1, 0);
         let recv_start = Instant::now();
-        let mms = verified_receiver.recv_timeout(timer)?;
+        let mut mms = verified_receiver.recv_timeout(timer)?;
+        // Try receiving more packets from verified_receiver. Let's coalesce any packets
+        // that are received within 20 ms of each other.
+        while let Ok(mut nq) = verified_receiver.recv_timeout(Duration::from_millis(20)) {
```

Comment:
How about a test that sends this a packet every 10ms? That or use `try_recv()`  ;)

Reply:
try_recv does not solve the problem. I'll see if a test can be added. Would it work if I file a ticket for the test and tackle it as a separate PR?

Reply:
Sorry, but I don't why this code wouldn't result in an OOM crash if you continued to send it a packet every 10ms.

Reply:
That's a good point. We can bound the loop with a maximum iteration count.